bin/fuzz.py

```diff
@@ -55,8 +55,7 @@ def _run(self, bar):
         localbar = bar.start(f'{self.i}: generate')
         result = self.generator.run(bar, cwd, name, self.seed)
```

Comment:
Should we pass `localbar` here? (not rhetoric)

Reply:
Oh huh, that is a good question :thinking: `bar.start` returns a copy, so probably yes?

Reply:
Fixed in #327: https://github.com/RagnarGrootKoerkamp/BAPCtools/pull/327/files#diff-445d79520fde92e79b3572d53d445cf7c57b3d3c58e98aef451519a2c2c766beR56